site/docs/master/sonobuoy-config.md

```diff
@@ -54,14 +54,45 @@ Filters
    - A Kubernetes [label selector][labelselector] which will be added to every query run.
 
 Limits
- - Options for limiting limits the scope of response when getting logs from pods. These will be passed onto Kubernetes [PodLogOptions][podlogopts]
- - The supported parameters are:
-    - Previous
-    - SinceSeconds
-    - SinceTime
-    - Timestamps
-    - TailLines
-    - LimitBytes
+ - Options for limiting limits the scope of response.
+ - **Limits.PodLogs** limits the scope when getting logs from pods. 
+     - The scope of pods is defined by:
+       ```
+       Namespaces OR SonobuoyNamespace OR FieldSelectors) AND LabelSelector
```

Comment:
I think you meant to start this with an `(`

Reply:
got it